src/react/src/features/auth/Register.tsx

```diff
@@ -26,58 +26,80 @@ export function Register() {
     }
 
     return (
-        <Box component="form" autoComplete="off" onSubmit={handleSubmit}
-            sx={{
-                display: 'flex',
-                flexDirection: 'column',
-                alignItems: 'center',
-                mt: 4
-            }}>
-            <TextField
-                label="First Name"
-                placeholder="Please enter your first name"
-                value={fname}
-                onChange={(e) => handleInput(e, setFname)}
-                margin="normal"
-                required
-                error={status === 'failed'}
-            />
-            <TextField
-                label="Last Name"
-                placeholder="Please enter your last name"
-                value={lname}
-                onChange={(e) => handleInput(e, setLname)}
-                margin="normal"
-                required
-                error={status === 'failed'}
-            />
-            <TextField
-                label="Email"
-                placeholder="Please enter your email"
-                value={email}
-                type="email"
-                onChange={(e) => handleInput(e, setEmail)}
-                margin="normal"
-                required
-                error={status === 'failed'}
-            />
-            <TextField
-                label="Password"
-                placeholder="Please enter your password"
-                value={password}
-                type="password"
-                onChange={(e) => handleInput(e, setPassword)}
-                margin="normal"
-                required
-                error={status === 'failed'}
-            />
-            <Button
-                type="submit"
-                variant="contained"
-            >
-                Sign Up
-            </Button>
-            {status === "failed" && message && <Alert severity="error">{message}</Alert>}
+        <Box sx={{height: '100vh', display: 'flex', alignItems: 'center', justifyContent: 'center'}}>
```

Comment:
See my comment on `Login.tsx`

Reply:
We should probably unify the styles so we don't have pixels, rems and vh all together